test/apiv2/80-kube.at

```diff
@@ -79,3 +79,71 @@ t DELETE libpod/play/kube $YAML 200 \
 rm -rf $TMPD
 
 # vim: filetype=sh
+
+# check kube play works when uploading body as a tar
+TMPD=$(mktemp -d podman-apiv2-test-kube.XXXXXX)
+KUBE_PLAY_TAR="${TMPD}/kubeplay.tar"
+cat > $TMPD/play.yaml << EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: demo-tar-remote
+spec:
+  containers:
+    - name: container
+      image: docker.io/library/nginx:latest
```

Comment:
please do not hard code random images, the image is unnecessary big which causes flakes during pulls, instead use `$IMAGE`. This is also why CI is failing as we do not have this image cached.

Reply:
I replaced with `$IMAGE` but the tests are failing on my machine when using it. Hope it will work for the CI